apps/web/src/components/search/index.js

```diff
@@ -20,16 +20,31 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 import * as Icon from "../icons";
 import "./search.css";
 import Field from "../field";
+import { useStore } from "../../stores/theme-store";
+import { getDarkScheme } from "@notesnook/theme/dist/theme/colorscheme/dark";
 
 function SearchBox(props) {
+  const { theme } = useStore();
+  const {text} = getDarkScheme()
   return (
     <Field
       data-test-id="search-input"
       autoFocus
       id="search"
       name="search"
       type="text"
-      sx={{ m: 0, mx: 1, mt: 1 }}
+      sx={{
+        m: 0,
+        mx: 1,
+        mt: 1,
+        ...(theme === "dark" && {
+          "input:-webkit-autofill": {
+            WebkitTextFillColor: text,
+            caretColor: "white",
```

Comment:
Instead of `white` use `text` here as well.

Reply:
ok